angular-material/angular-material-tests.ts

```diff
@@ -4,7 +4,8 @@ var myApp = angular.module('testModule', ['ngMaterial']);
 
 myApp.config((
     $mdThemingProvider: ng.material.IThemingProvider,
-    $mdIconProvider: ng.material.IIconProvider) => {
+    $mdIconProvider: ng.material.IIconProvider,
+	$mdProgressCircularProvider: ng.material.IProgressCircularProvider) => {
```

Comment:
Fixed. However i have tried to rebase this pull request with `master` and i have got some merge conflicts. Is the first rebase for me :)